engine/class_modules/priest/sc_priest.cpp

```diff
@@ -559,11 +570,20 @@ struct smite_t final : public priest_spell_t
   {
     priest_spell_t::execute();
 
-    if ( priest().talents.manipulation.enabled() &&
-         ( priest().specialization() == PRIEST_HOLY || priest().specialization() == PRIEST_DISCIPLINE ) )
```

Comment:
This is actually not true based on the tooltip 
![image](https://user-images.githubusercontent.com/10604059/224524238-cd9b7c0b-3986-4887-8283-9e9183bfa205.png)

Reply:
Added these safety checks back in here, as well as for mind blast: https://github.com/simulationcraft/simc/pull/7413/files/3dfbcd0b7ae91b4468f581ad406ba05348644fda#diff-3642d3768d1526e985a4c06a468c29788411f96429999aa347a669aa7c32d9adL90